plugins/modules/database/saphana/hana_query.py

```diff
@@ -0,0 +1,208 @@
+#!/usr/bin/python
+
+# Copyright: (c) 2021, Rainer Leber <rainerleber@gmail.com>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+from __future__ import (absolute_import, division, print_function)
+__metaclass__ = type
+
+DOCUMENTATION = r'''
+---
+module: hana_query
+
+short_description: Execute SQL on HANA.
+
+version_added: "3.2.0"
+
+description: This module executes SQL statements on HANA with hdbsql.
+
+options:
+    sid:
+        description: The system ID.
+        type: str
+    instance:
+        description: The instance number.
+        type: str
+    user:
+        description: A dedicated username. Defaults to SYSTEM.
+        type: str
+        default: SYSTEM
+    password:
+        description: The password to connect to the database.
+        type: str
+    autocommit:
+        description: Autocommit the statement.
+        type: bool
+        default: True
+    host:
+        description: The Host ip address. The port can be defined as well.
+        type: str
+    database:
+        description: Define the database on which to connect.
+        type: str
+    encrypted:
+        description: Use encrypted connection. Defaults to false.
+        type: bool
+        default: false
+    filepath:
+        description:
+        - SQL query in file to run. Multiple files can be passed using YAML list syntax.
```

Comment:
```suggestion
        - One or more files each containing one SQL query to run.
```
(Or can there be multiple per file?)